misc/sampleapp-android-tests/sampleapp/M42_AppInstallBanners.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Xu, Kang <kangx.xu@intel.com>
+
+-->
+
+<meta charset="utf-8">
+<title>Chrome42_App Install Banners</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<div>
+  <p>Test purpose:</p>
+  <ol>
+    <li>Note: the sample need network support.</li>
+    <li>Check if App Install Banners is supported or not.</li>
+  </ol>
+  <p>Test steps:</p>
+  <ol>
+    <li>Open the url "https://googlechrome.github.io/samples/app-install-banner/index.html"</li>
+    <li>Come back in anthor day, check the home screen</li>
+  </ol>
+  <p>Expected result:</p>
+  <p>The banner for sites which is visitted frequently should be added to home screen</p>
```

Comment:
The expected result shall be a comparison of the sample running on Crosswalk and on Chrome. Please update them all.